Acrylic card

Virrum opened the pull request:
I have some issues onwriting tests on the magic that happens in aurelia lifecycle. Need help from @einari  and have created a separate task for that.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1115616103797671/1153138530362504)

Commit:
Set the behavior of the Acrylic card

Commit:
writing a failing test, needs fix from a hero

Commit:
Acrylic card documentation

Commit:
Fixed typos

Commit:
restructured style of gallery

einari commented on Source/AcrylicCard/for_AcrylicCard/when_initiating_an_acrylic_card.ts:
```diff
@@ -0,0 +1,29 @@
+import { bootstrap } from 'aurelia-bootstrapper';
+import { AcrylicCard } from '../AcrylicCard';
+
+describe('When initiating an acrylic card', () => {
```
Consistency; lowercase **When** - we have all our specs in C# and TS/JS already in this way

Commit:
Added a base Component class

Commit:
more usage tests in Gallery

Commit:
Fixed Acrylic card to how it should be done

einari committed:
We've consistently used lower case for the "when" statement

This will also make it consistent with the file. We can of course choose to display this in a report differently, as having a sentence start with a lower case doesn't really make sense. So, torn on this - but for consistency lets keep doing one thing.

einari reviewed (approved):
This looks great - and the PR consisting of all aspects of what a Component is; including documentation and sample.
